students/daniel_carrasco/session09/cli_main.py

```diff
@@ -0,0 +1,134 @@
+import donor_models as dm
+import donor_database as ddb
+import os
+import sys
+import pytest
+
+donors = {'Art Bart': [1000, 1], 'Harry Scary': [50, 5], 'Hay Boo': [50000, 3]}
+for keys in donors:
+    dm.Donor(keys).donations = donors[keys][0]
+    dm.Donor(keys).counter = donors[keys][1]
+
+path = os.getcwd()
+
+
+def main():
+
+    while True:
+        try:
+            user_choice = int(input('\nChoose an action(1 - 4):\n\n\
+                    1 - Send a Thank You to a single donor.\n\
+                    2 - Create a Report.\n\
+                    3 - Send letters to all donors.\n\
+                    4 - Add new donor.\n\
+                    5 - Add new donation.\n\
+                    6 - Quit\n'))
+            arg_dict = {
+                1: thankyou,
+                2: report,
+                3: letter,
+                4: addnew,
+                5: addnew,
+                6: quit}
+
+            if arg_dict[user_choice] == 'quit':
+                sys.exit()
```

Comment:
you can actually get rid of this `if` statement if you keep your `quit` without the quotes and it will execute `quit()` which is a python built in and will do the same as `sys.exit()`